webapp/managers.py

```diff
@@ -244,6 +256,48 @@ def create_table(
             rows.append([id, time, gt, task, variant, ip, code])
         return rows
 
+    def create_exam_result_table(
+        self,
+        group_id: int,
+        statuses: TaskStatusRepository,
+        variants: VariantRepository
+    ) -> List[List[str]]:
+        result_table = []
+        headings = [
+                    'Сдающая_группа', 'Вариант', '№1_Группа',
+                    '№1_Вариант', '№1_Задача', '№1_Статус',
+                    '№2_Группа', '№2_Вариант', '№2_Задача',
+                    '№2_Статус', 'Сумма_баллов'
+                   ]
+        result_table.append(headings)
```

Comment:
Есть мнение, что данный код можно упростить до:
```suggestion
        result_table = [[
            'Сдающая_группа', 'Вариант', '№1_Группа',
            '№1_Вариант', '№1_Задача', '№1_Статус',
            '№2_Группа', '№2_Вариант', '№2_Задача',
            '№2_Статус', 'Сумма_баллов'
        ]]
```